spirv_msl.hpp

```diff
@@ -266,6 +266,7 @@ class CompilerMSL : public CompilerGLSL
 		uint32_t dynamic_offsets_buffer_index = 23;
 		uint32_t shader_input_wg_index = 0;
 		uint32_t device_index = 0;
+		uint32_t argument_buffer_offset = 0; // Allow the caller to specify an offset for argument buffer binding slots
```

Comment:
This should not be needed. Are you using the set_msl_resource_binding API at all?

Reply:
I think the `add_msl_resource_binding` API isn't used in our pipeline yet, maybe because it was added after the changes for UE4 were made. I removed this parameter and will add your recommended API on our side.